packaging/aznfs/RPM/aznfs.spec

```diff
@@ -90,6 +87,22 @@ if ! command -v stunnel >/dev/null 2>&1 || (( $(echo "$REQUIRED_STUNNEL_VERSION
 	fi
 
 	cleanup_stunnel_files
+
+	# Remove the old link and create a symlink to stunnel binary.
+    [ -f /bin/stunnel ] && mv /bin/stunnel /bin/stunnel.old
+    rm -f /bin/stunnel
```

Comment:
forgot to address this?

Reply:
Sorry, I forgot about this one. I'd only removed it from postint.